explorer/app/config/model.ts

```diff
@@ -1,5 +1,34 @@
+import {
+  DetailResponseType,
+  DetailViewModel,
+  ListResponseType,
+  ListViewModel,
+} from "app/models";
 import { HeaderProps } from "../components/Header/Header";
 
+type ListTransformerFunction<T extends ListResponseType> = (
+  response: T
+) => ListViewModel;
+
+type DetailTransformerFunction<T extends DetailResponseType> = (
+  response: T
+) => DetailViewModel;
+
+type GetIdFunction<T extends DetailResponseType> = (detail: T) => string;
+
+export interface EntityConfig<
+  L extends ListResponseType = any,
```

Comment:
Two questions here:

1. Why do we need to default L and D to `any`?
2. Why is the linter not picking up the usage of `any` here? 

Thanks!